minorminer/utils/raster_embedding.py

```diff
@@ -70,19 +70,37 @@ def visualize_embeddings(H, embeddings=None, **kwargs):
         'width': 1,
         'cmap': cmap,
         'edge_cmap': cmap,
+        'node_size': 300/np.sqrt(H.number_of_nodes())
     }
     draw_kwargs.update(kwargs)
 
     topology = H.graph.get('family') 
     # Draw the combined graph with color mappings
     if topology == 'chimera':
+        pos = dnx.chimera_layout(H)
         dnx.draw_chimera(**draw_kwargs)
     elif topology == 'pegasus':
-        dnx.draw_pegasus(**draw_kwargs, crosses=True)
+        pos = dnx.pegasus_layout(H)
+        dnx.draw_pegasus(**draw_kwargs)
     elif topology == 'zephyr':
+        pos = dnx.zephyr_layout(H)
         dnx.draw_zephyr(**draw_kwargs)
     else:
+        pos = nx.spring_layout(H)
         nx.draw_networkx(**draw_kwargs)
+
+    # Recolor specific edges on top of the original graph
+    highlight_edges = [e for e in H.edges() if not np.isnan(edge_color_dict[e])]
+    highlight_colors = [edge_color_dict[e] for e in highlight_edges]
+    
+    nx.draw_networkx_edges(
```

Comment:
In this new flow with two calls to draw, it is not necessary to color anything (other than grey) on the first call. We could consider simplifying the call arguments. This can be done later, needn't hold up the pull request.